apps/ledger-live-desktop/src/renderer/screens/dashboard/GlobalSummary.tsx

```diff
@@ -24,6 +24,7 @@ export default function PortfolioBalanceSummary({ range, chartColor, counterValu
     (val: number | string) => formatShort(counterValue.units[0], BigNumber(val)),
     [counterValue],
   );
+  const suggestedMin = Math.max(Math.min(...portfolio.balanceHistory.map(b => b.value)), 0);
```

Comment:
could you rework this with a reduce instead? It is generally considered unsafe to spread arguments on min / max because it can fails on certain array size.
( ref https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max )

![Capture d’écran 2024-03-25 à 10 39 22](https://github.com/LedgerHQ/ledger-live/assets/211411/ac34f990-9512-4220-b970-48f6f247e1a8)